### ISSUE TITLE
1.21 Compatibility 

### ISSUE THREAD

**Issue by RealRagnvald** [mcbe-mods/Cut-tree-one-click#11]
Will this be compatible with 1.21 on June 13th or do you plan on updating it to work with the version?

**Comment by lete114** [comment]
That's if I'm available.

**Comment by RealRagnvald** [comment]
It's stable as in it won't break anymore?

**Comment by lete114** [comment]
Yes, it will be stable after 1.21, 1.22, 1.23, 1.24. will still be available

**Comment by albertethon** [comment]
it seems not work at 1.21.7  ?

**Comment by lete114** [comment]
> it seems not work at 1.21.7 ?

Did you download the latest mcpack?
https://github.com/mcbe-mods/Cut-tree-one-click/releases/tag/v1.0.8

If v1.0.8 really does not work, then I will find time to adapt
